starfish/experiment/builder/cli.py

```diff
@@ -47,8 +48,9 @@ def dimensions_option(name, required):
 
 def build(output_dir, fov_count, hybridization_dimensions, **kwargs):
     write_experiment_json(
-        output_dir, fov_count, hybridization_dimensions,
-        kwargs
+        output_dir, fov_count, ImageFormat.TIFF,
```

Comment:
is TIFF the default?

Reply:
I am declaring it so. :)

I suspect it makes sense because it allows for a quick visualization of the stack after building the experiment object.  Any thoughts to the contrary?